cache/lru_cache.cc

```diff
@@ -467,6 +467,7 @@ Cache::Handle* LRUCacheShard::Lookup(
 
       if (wait) {
         Promote(e);
+        e->SetIsInSecondaryCache(is_in_sec_cache);
```

Comment:
Need to set this in the else part too. That was my main concern in my earlier comment.